manifests/mount.pp

```diff
@@ -9,7 +9,8 @@
   $cvmfs_max_ttl = undef,
   $cvmfs_env_variables = undef,
   $cvmfs_use_geoapi = undef,
-  $cvmfs_repo_list = true
+  $cvmfs_repo_list = true,
+  $cmvfs_mount_rw = 'no',
```

Comment:
@traylenator Yes, this should definitely be undef too like in params.pp. Sorry, apparently I switched from 'no' to undef in params.pp file only in my PR. I forgot about mount.pp, which is not using the params.pp to set it's defaults.

I'll change this, rebase and test again. Ok?